apps/client/src/views/timer/Timer.tsx

```diff
@@ -177,7 +177,7 @@ function Timer({ customFields, projectData, isMirrored, settings, viewSettings }
           </FitText>
         ) : (
           <div
-            className={cx(['timer', !isPlaying && 'timer--paused', showFinished && 'timer--finished'])}
+            className={cx(['timer', !isPlaying && 'timer--paused', showFinished && 'timer--finished', `timer-type--${viewTimerType}`])}
```

Comment:
Since we don’t use the class for anything. I think a data attribute may be more useful

On another hand, should we just hide the time if timer type is none? It was in our todo list